_posts/2020-11-01-batch-processing-stop-job-step.adoc

```diff
@@ -0,0 +1,229 @@
+---
+layout: post
+title: How to properly stop batch processing job and step in WildFly
+date: 2020-11-01
```

Comment:
@chengfang Please change the 'date' field here to the day you make this change or the day after, and please also change the name of the file to match. Once it's updated please ping me in zulip so I can get it merged.

Reply:
changed the date field and renamed the file.